src/figure/FigureStateActive.js

```diff
@@ -170,5 +173,14 @@ export default function (stateFactory, ActiveState, formTpl, formElement, inline
         }
     };
 
+    ImgStateActive.prototype.destroyEditor = function () {
+        const widget = this.widget;
+        const $editableContainer = widget.$original.parents('[data-qti-class="_container"]');
+        htmlEditor.buildEditor($editableContainer, {});
+        _.defer(() => {
+            htmlEditor.destroyEditor($editableContainer);
+        });
```

Comment:
**question:** Why building then destroying the editor immediately?

Reply:
The CKEditor reads the Figure HTML tag, and if it is inside a block, it moves out of it.
To achieve this effect, I must rebuild the editor, but I don't want to have it active, so I destroy it afterwards. Otherwise, I will have an error in the XML structure.